syntax = "proto3";

package limdongjin.ignasr.protos;

message SpeechRequest {
  string reqId = 1;
  string userId = 2;
  string videoId = 3;
  int32 numOfChunk = 4;
}
